CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.9.6
+
+* Revert #597 (#639)
+* Harden check when rendering media library (#652)
+* Clean comment cache after direct SQL queries. (#658)
```

Comment:
This line has a full stop, but the others don't?

Reply:
I am nothing if not incosistent. [a6f6e6ccd2c492a2e74d7dbe8043f1f08a6f7743]